### DISCUSSION TITLE
Can you replace webpack for rspack in an old nuxt 2 project?

### DISCUSSION THREAD

**Discussion by Nisthar** [web-infra-dev/rspack#7060] [Q&A]
i have a nuxt 2 application that i am working for a client. i am wondering if its possible to replace the webpack with rspack for performance boost.

**Comment by nehuenpereyra** [comment]
I am currently with the same problem (Nuxt v2.15.8)

**Comment by hardfist** [comment]
It seems nuxt2 reaches EOL, and nuxt3 is considering adopting Rspack https://github.com/nuxt/nuxt/pull/29142, if someone willing to make nuxt2 run on Rspack, we're willing to help but we don't have plan to do it ourself. also cc @danielroe
